.travis.yml

```diff
@@ -49,7 +50,7 @@ jobs:
       script:
         - echo "deploy-snapshots"
         # When push to master occurs, all the versions should be -SNAPSHOTs and we will auto-deploy
-        - mvn --batch-mode -e -DskipTests=true deploy
+        - mvn --batch-mode -e -DskipTests=true deploy -DrepositoryId=artifact-registry -Durl=https://us-west1-maven.pkg.dev/som-rit-infrastructure-prod/public-maven
```

Comment:
Like the other build, these properties don't look right. Might want to double check them.